src/components/App/App.tsx

```diff
@@ -478,10 +530,9 @@ const GrpcDocApplication: React.FC<GrpcPlaygroundApplicationProps> = ({ appId, s
     const isGenDoc = true;
 
     if (isGenDoc) {
-      let handlerId: string | undefined;
       // eslint-disable-next-line prefer-const
-      handlerId = addUploadedListener(protos => {
-        removeUploadedListener(handlerId!);
+      const handlerId = addUploadedListener(undefined, protos => {
+        removeUploadedListener(undefined, handlerId!);
 
```

Comment:
any reason why we provide `undefined` here to the listener?

Reply:
I removed it